### PR TITLE
[fix] grafana deploy create configmap check

### PR DESCRIPTION
**Pull request by YanjieGao**

### CODE REVIEW

File: pai-management/bootstrap/grafana/start.sh [hunk 1]
```diff
@@ -27,6 +27,13 @@ pushd $(dirname "$0") > /dev/null
 
 /bin/bash node-label.sh
 
+while !  kubectl get configmap | grep -q "grafana-configuration"
```
Comment (ydye) [inline pai-management/bootstrap/grafana/start.sh]: Have you tested this?

Comment (YanjieGao) [reply pai-management/bootstrap/grafana/start.sh]: tested for created and not created

Comment (YanjieGao) [reply pai-management/bootstrap/grafana/start.sh]: yes, tested

File: pai-management/bootstrap/grafana/start.sh [hunk 2]
```diff
@@ -27,6 +27,20 @@ pushd $(dirname "$0") > /dev/null
...
 /bin/bash node-label.sh
 
+i=0
+
+while !  kubectl get configmap | grep -q "grafana-configuration" 
+do
+  sleep 3
+  echo "grafana-configuration configmap does not exist, try to create grafana configmap"
+  /bin/bash configmap-create.sh
+  i=`expr $i + 1`;
```
Comment (sterowang) [inline pai-management/bootstrap/grafana/start.sh]: Will there more detailed error can get if configmap not able to created?

**Pull request merged**